src/FileConventions/Library.fs

```diff
@@ -396,5 +396,16 @@ let DefiningEmptyStringsWithDoubleQuotes(fileInfo: FileInfo) =
     fileText.Contains "\"\""
 
 let ProjFilesNamingConvention(fileInfo: FileInfo) =
-    printfn "%s" fileInfo.FullName
-    false
+    let regex = new Regex("(.*)\..*proj$")
+    assert (regex.IsMatch fileInfo.FullName)
```

Comment:
not sure about this but I think paren is not needed

Reply:
it is needed; because `assert` doesn't receive 2 currified arguments

Reply:
assert is for debug mode only, are we sure it should be used here?

Reply:
> it is needed; because `assert` doesn't receive 2 currified arguments

Not sure because it compiles without them :thinking:

Reply:
@aarani you're right, they should use Fsdk's Misc.BetterAssert